macros/get_ad_squad_hourly_report_columns.sql

```diff
@@ -0,0 +1,49 @@
+{% macro get_ad_squad_hourly_report_columns() %}
```

Comment:
For each of the macros, you will need to remove the columns that are not used in the final CTE of the staging model. Can you go through the macros and make sure we are only bringing through the relevant fields.